packages/protocol/contracts/layer1/core/iface/IInbox.sol

```diff
@@ -72,6 +73,8 @@ interface IInbox {
         bytes32 originBlockHash;
         /// @notice The percentage of base fee paid to coinbase.
         uint8 basefeeSharingPctg;
+        /// @notice The liveness bond amount reserved for this proposal at propose time.
+        uint256 livenessBond;
```

Comment:
Let's also make this part of the `Proposed` event payload? Would be great for off-chain software to track.